src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
your can avoid code duplicates and write these selectors with coma like this
```css
.stars--1 > :nth-child(1),
.stars--2 > nth-child(-n+2),
// other selectors {
background-image: // ...
}
```

Reply:
Hi @anastasiia-tilikina , wrote selectors with commas